beamer/tests/agent/test_challenge.py

```diff
@@ -408,3 +409,92 @@ def test_challenge_7(request_manager, fill_manager, token, config):
 
     assert agent_earnings() == stake
     assert dave_earnings() == stake + 1
+
+
+@pytest.mark.usefixtures("setup_relayer_executable")
+def test_l1_resolution(request_manager, fill_manager, token, config, direction, chain_params):
```

Comment:
why not reusing the agentfixture?

Reply:
agent is using ganache account which i cant drain really, in this test i wanted to have an agent with limited funds.

Reply:
as agent fixture is working in my current trial i am updating to use agent fixture here.